packages/xo-server/src/xo-mixins/migrate-vm.mjs

```diff
@@ -153,7 +153,7 @@ export default class MigrateVm {
     return chainsByNodes
   }
 
-  #connectToEsxi(host, user, password, sslVerify = true) {
```

Comment:
Why this change?

Reply:
default value was set in multiple places ( in api and here)  maybe ot should be a separate commit ?

Reply:
Don't bother, IMHO this PR is simple enough to understand this is unrelated.